Cubical/HITs/Pushout/Properties.agda

```diff
@@ -16,13 +19,166 @@ open import Cubical.Core.Glue
 open import Cubical.Foundations.Prelude
 open import Cubical.Foundations.Equiv
 open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.HAEquiv
 open import Cubical.Foundations.GroupoidLaws
 
 open import Cubical.Data.Prod.Base
 open import Cubical.Data.Unit
 
 open import Cubical.HITs.Pushout.Base
 
+{-
+  Definition of pushout diagrams
+-}
+
+record 3-span : Type₁ where
+  field
+    A0 A2 A4 : Type₀
+    f1 : A2 → A0
+    f3 : A2 → A4
+
+spanPushout : (s : 3-span) → Type₀
+spanPushout s = Pushout (3-span.f1 s) (3-span.f3 s)
+
+{-
+  Definition of a homotopy natural diagram equivalence
+-}
+
+record 3-span-equiv (s1 : 3-span) (s2 : 3-span) : Type₀ where
+   field
+     e0 : 3-span.A0 s1 ≃ 3-span.A0 s2
+     e2 : 3-span.A2 s1 ≃ 3-span.A2 s2
+     e4 : 3-span.A4 s1 ≃ 3-span.A4 s2
+     H1 : ∀ x → 3-span.f1 s2 (e2 .fst x) ≡ e0 .fst (3-span.f1 s1 x)
+     H3 : ∀ x → 3-span.f3 s2 (e2 .fst x) ≡ e4 .fst (3-span.f3 s1 x)
+
+{-
+  Proof that the pushouts of equivalent diagrams are equal
+-}
+
+spanEquivToPath : {s1 : 3-span} → {s2 : 3-span} → (e : 3-span-equiv s1 s2)
```

Comment:
Hmm, I have a feeling that there must be a more direct proof of this using Glue types... But maybe we can investigate that later.

Reply:
Yes, retrospectively this proof is a bit silly since it does not even use univalence (except in the very end). I am pretty sure I could do it with some simple transport.

Reply:
I could indeed do it in a much shorter manner, and it barely took 1 hour. Now I feel stupid for spending so much time on this. Oh well, a reminder to take univalence seriously is always good :)

Reply:
Can you push the new proof?

Reply:
Done. Despite the old one not being totally devoid of interest (it does the job in a non-univalent world), I don't think there is much of a point to keep it around, so I just removed it from the PR.